src/i18n/zh-cn/nav.ts

```diff
@@ -68,7 +68,7 @@ export default NavDictionary({
 	'reference/integrations-reference': '集成 API',
 	'reference/adapter-reference': '适配器 API',
 	'reference/image-service-reference': '图像服务 API',
-	'reference/dev-overlay-plugin-reference': 'Dev Overlay 插件 API',
+	'reference/dev-toolbar-app-reference': '开发工具栏插件 API',
```

Comment:
```suggestion
	'reference/dev-toolbar-app-reference': '开发工具栏应用 API',
```

From suggestion by @100gle